monitoring/settings.py

```diff
@@ -25,8 +31,37 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+try:
+    ALLOWED_HOSTS = cp.get('common', 'allowed_hosts').split(',')
 
+    # Database
+    # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.sqlite3',
+            'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        },
+        'grid': {
+            'ENGINE': cp.get('db_grid', 'backend'),
+            'HOST': cp.get('db_grid', 'hostname'),
+            'PORT': cp.get('db_grid', 'port'),
+            'NAME': cp.get('db_grid', 'name'),
+            'USER': cp.get('db_grid', 'username'),
+            'PASSWORD': cp.get('db_grid', 'password'),
+        },
+        'cloud': {
+            'ENGINE': cp.get('db_cloud', 'backend'),
+            'HOST': cp.get('db_cloud', 'hostname'),
+            'PORT': cp.get('db_cloud', 'port'),
+            'NAME': cp.get('db_cloud', 'name'),
+            'USER': cp.get('db_cloud', 'username'),
+            'PASSWORD': cp.get('db_cloud', 'password'),
+        },
+    }
+
+except Exception as err:
```

Comment:
Really don't like catch-all handling.

The main thing to catch is `configparser.NoSectionError` as that's what gets raised if the settings file is missing - add a message to suggest checking that the settings.ini exists. Anything else should just be allowed to raise itself.

Reply:
Added base class for `configparser` Errors.

Reply:
How come you didn't just catch the exception I listed?

Reply:
Added configparser.Error as a base class to handle all errors. If it the intension to break the system. once call pass duplicates or can do anything in the configuration file. Which is why I thought to add base class.

Just like file missing from AQ and the permission issues. Please let me know your views on that, Adrian

Reply:
Or

We can just have OSError to handle the file (settings.ini). And, Leave the rest about what goes inside the file we can rely on AQ. Since AQ does that check for us.

Reply:
The point is that most Python exceptions give you a fairly clear idea of what went wrong, so you only need to specifically handle them if you know there's a way to work around the exception. However, if a config file is missing, then `configparser` actually raises a `NoSectionError` when you try to retrieve a config value, which isn't particularly clear as to what's actually gone wrong.

Reply:
Cool. Updated the code to handle the file or section header missing case using `NoSectionError` from `configparser` .